### PR TITLE
Few additions

### PR DESCRIPTION
**Pull request by dpeterka**

### CODE REVIEW

File: documentation/troubleshooting.md [hunk 1]
```diff
@@ -10,6 +10,56 @@ lang: en
...
+
+### Logs
+
+#### Examine upstart logs
+
+Review logs in `/var/log/upstart/{service-name}.log`.  Spinnaker services are: clouddriver, rosco, rush, igor, front50, gate and orca.
+
+#### Examine application logs
+
+Review logs in `/var/log/spinnaker/{service-name}/{service-name}.log
```
Comment (robfletcher) [inline documentation/troubleshooting.md]: Looks like it's missing a closing backtick there

**Comment by tomaslin** [comment]
ping @spinnaker/reviewers

File: documentation/troubleshooting.md [hunk 2]
```diff
@@ -10,6 +10,56 @@ lang: en
...
+
+### Ensure permissions
+
+#### AWS
+
+Ensure you have a API access key and secret correctly configured in /home/spinnaker/.aws/credentials or an EC2 Instance role with "PowerUser" policy attached to the Spinnaker instance.
```
Comment (anotherchrisberry) [inline documentation/troubleshooting.md]: `a API access key` -> `an API access key`

File: documentation/troubleshooting.md [hunk 3]
```diff
@@ -10,6 +10,56 @@ lang: en
...
+
+#### Examine application logs
+
+Review logs in `/var/log/spinnaker/{service-name}/{service-name}.log`
+
+### Ensure permissions
+
+#### AWS
+
+Ensure you have an API access key and secret correctly configured in /home/spinnaker/.aws/credentials or an EC2 Instance role with "PowerUser" policy attached to the Spinnaker instance.
```
Comment (ewiseblatt) [inline documentation/troubleshooting.md]: The PowerUser policy is only applicable if spinnaker is deployed on AWS.

Comment (tomaslin) [inline documentation/troubleshooting.md]: the heading does say AWS

Comment (ewiseblatt) [inline documentation/troubleshooting.md]: I took it to mean configuring to access AWS not deploy onto AWS. The .aws/credential instructions are relevant to configuring AWS irregardless of where spinnaker is deployed.

**Comment by anotherchrisberry** [comment]
@dpeterka @ewiseblatt @tomaslin is this PR still relevant or should it be closed?

**Comment by dpeterka** [comment]
+ @duftler I updated the branch w/ some suggestions not sure if we these have been addressed elsewhere by now or not.

**Comment by tomaslin** [comment]
@duftler ping

File: documentation/troubleshooting.md [hunk 4]
```diff
@@ -10,6 +10,56 @@ lang: en
...
+#### Apache
+
+Execute `curl -I localhost:9000`. Response should be a HTTP code 200.
+
+
+### Logs
+
+#### Examine upstart logs
+
+Review logs in `/var/log/upstart/{service-name}.log`.  Spinnaker services are: clouddriver, rosco, rush, igor, front50, gate and orca.
```
Comment (duftler) [inline documentation/troubleshooting.md]: And echo.

File: documentation/troubleshooting.md [hunk 5]
```diff
@@ -10,6 +10,56 @@ lang: en
...
+
+#### Examine application logs
+
+Review logs in `/var/log/spinnaker/{service-name}/{service-name}.log`
+
+### Ensure permissions
+
+#### AWS
```
Comment (duftler) [inline documentation/troubleshooting.md]: Seems like we've covered this particular item in detail here (http://spinnaker.io/documentation/getting_started.html#step-1-set-up-your-target-deployment-environment). Can we just link to that from here? If something is missing, we can add it there. WDYT?

**Comment by tomaslin** [comment]
@dpeterka - ping

**Comment by dpeterka** [comment]
added echo. @duftler I am not sure where you were referencing the link to be added. I am not sure how much more we want to spend on this particular PR I am pondering just taking a re-write stab at things . . .

**Comment by duftler** [comment]
I was suggesting to drop the content in the `Ensure permissions` section entirely and just link to the relevant section in the Getting Started guide instead.

I think the pr is useful enough that we should try to finish it up and get it merged in (can always pursue a rewrite afterwards).

Also looks like the base troubleshooting guide has had a couple of items added in the meantime.

**Comment by ajordens** [comment]
Last commented 28 days ago ... where does this sit?

**Comment by tomaslin** [comment]
Going to close this issue and we can add the additions once we migrate over to readme.io

**Pull request closed without merging**